course_viewer/course_viewer/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+
+class State(models.Model):
+    '''
+    The Transfer State
+    '''
+    state_code = models.CharField(max_length=2,
+                                  help_text='',
+                                  unique=True)
+    state_name = models.CharField(max_length=200,
+                                  unique=True)
+
+
+class College(models.Model):
+    '''
+    The Transfer College
+    '''
+    college_code = models.CharField(max_length=15,
+                                    help_text='',
+                                    unique=True)
+    college_name = models.CharField(max_length=200,
+                                    unique=True)
+
+
+class Course(models.Model):
+    '''
+    The Transfer Course
+    '''
+    course_name = models.CharField(max_length=200)
+    course_credit = models.IntegerField()
+    course_number = models.IntegerField()
```

Comment:
Course numbers aren't always number integers.  For example in our current CSV data on row 77 we have values such as `102HC`  as "course_number".  Perhaps course_id would be more appropriate?  Will probably change that in the script on my next PR.